src/server/swagger_docs/vscode.dev_powerplatform_v2.0.yml

```diff
@@ -0,0 +1,1360 @@
+openapi: "3.0.2"
```

Comment:
just verify this file for accuracy and add this to repo

Reply:
done